Microsoft.Identity.Web/README.md

```diff
@@ -9,6 +9,14 @@ This library contains a set of reusable classes useful in ASP.NET Core:
 
 to enable them to work with the Microsoft identity platform (formerly named Azure AD v2.0 endpoint). In the library, web apps and protected web APIs are collectively referred to as web resources.
 
+
+ ## Breaking changes
+
+ If you've been using Microsoft.Identity.Web in your project you might want to know that, on 12/05/2019, a number of APIs are renamed to be consistent:
+.AddMicrosoftIdentityPlatformAuthentication => AddSignIn
+.AddMsal => .AddWebAppCallsProtectedWebApi
```

Comment:
>AddWebAppCallsProtectedWebApi [](start = 13, length = 29)

I would also propose we should not hesitate using the term “on behalf of” as it satisfies developers who read docs or blogs abt using the on behalf of flow for their scenario and come looking for an useful and easy implementation